src/Schema/Event.ts

```diff
@@ -93,3 +105,38 @@ export interface SuccessfullyLoggedIn {
   auth_redirect: string
   service: AuthService
 }
+
+/**
+ * A user taps a rail, card, or other grouping of entities on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedEntityGroup"
+ *    context_module: "newWorksByArtistsYouFollowRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Artwork
+ *    destination_screen_owner_id: "5e9a7a238483bf000e2c4c5e"
+ *    destination_screen_owner_slug: "romain-jacquet-lagreze-makeshift-garden-hong-kong"
+ *    horizontal_slide_position: 1
+ *    module_height: "single"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedEntityGroup {
+  action: ActionType.tappedEntityGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  freshness?: number
```

Comment:
pitching `freshness` to describe how old the works tapped are on the [new works-for-you screen](https://www.figma.com/file/gzAi9BJi62AaJuhMRsgXMj/Home?node-id=196%3A817): 
<img width="300" alt="Screen Shot 2020-04-23 at 3 03 06 PM" src="https://user-images.githubusercontent.com/26879583/80138908-9671e000-8573-11ea-9996-642fe0c7c584.png">

For example, if you tap today's works, your `freshness` = 0, but if you tap yesterday's, it's 1. The number should be the count of days between the current_date and when the works were added to the user's feed.

Reply:
We're holding on this screen b/c I think the freshness we were trying to have here requires a heavier lift on our back-end. CC @ashfurrow

Reply:
gotcha. we can keep out of the event for now then!